src/components/Combobox/Combobox.tsx

```diff
@@ -216,16 +217,25 @@ function Combobox<Item>({
                   {label}
                 </InputLabel>
               </InputControl>
-              <Icon
+              <Flex
                 opacity={disabled ? 0.3 : 1}
-                type={isOpen ? 'caret-up' : 'caret-down'}
                 position="absolute"
-                pointerEvents="none"
-                my="auto"
                 top={0}
                 bottom={0}
-                right={4}
-              />
+                right={2}
+                align="center"
+                justify="center"
+                pointerEvents={isOpen ? 'auto' : 'none'}
+              >
+                <IconButton
+                  tabIndex={-1}
+                  icon={isOpen ? 'caret-up' : 'caret-down'}
+                  size="medium"
+                  aria-label="Toggle Menu"
+                  onClick={() => closeMenu()}
+                  variant="unstyled"
```

Comment:
how come the unstyled variant has a background? Is that a bug?

Reply:
I was asking myself the same thing. I tried a lot of different prop combinations in order to find a way to have a transparent background but i couldn't find one that works. I thought that you did that on purpose and it has to do with accessibility/usability compliance stuff...

Reply:
Can't remember. Could be 😆